include/threads/ThreadState.h

```diff
@@ -87,16 +108,10 @@ class OpenMPTask
     {}
 };
 
-class OpenMPContext
-{
-  public:
-    int threadNumber = -1;
-    std::shared_ptr<Level> level = nullptr;
-};
```

Comment:
There was no need for this wrapper object, the only thing that's actually shared is the `Level`